gen/rollup/v1/helpers.go

```diff
@@ -0,0 +1,34 @@
+package rollupv1
+
+import (
+	"errors"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+)
+
+var _ sdktypes.Msg = (*ApplyL1TxsRequest)(nil)
+
+func (*ApplyL1TxsRequest) GetSigners() []sdktypes.AccAddress {
+	return nil
+}
+
+func (m *ApplyL1TxsRequest) ValidateBasic() error {
+	if len(m.TxBytes) < 1 {
+		return errors.New("expected TxBytes to contain at least one deposit transaction")
+	}
+	return nil
+}
+
+func (*ApplyL1TxsRequest) Type() string {
+	return "l1txs"
+}
+
+func (*ApplyL1TxsRequest) Route() string {
+	return "rollup"
+}
```

Comment:
I think these are unused?

Reply:
Correct. They are used to fulfill interfaces that the cosmos-sdk requires.